test/models/post_test.exs

```diff
@@ -14,76 +14,98 @@ defmodule CodeCorps.PostTest do
     status: "nonexistent"
   }
 
-  test "create changeset with valid attributes is valid" do
-    user = insert(:user)
-    project = insert(:project)
-    changeset = Post.create_changeset(%Post{}, %{
-      markdown: "some content",
-      post_type: "issue",
-      title: "some content",
-      project_id: project.id,
-      user_id: user.id,
-      status: "open"
-    })
-    assert changeset.valid?
+  describe "&create/2" do
+    test "is invalid with invalid attributes" do
+      changeset = Post.changeset(%Post{}, @invalid_attrs)
+      refute changeset.valid?
+    end
+
+    test "only allows specific values for post_type" do
+      changes = Map.put(@valid_attrs, :post_type, "nonexistent")
+      changeset = Post.changeset(%Post{}, changes)
+      refute changeset.valid?
+    end
+
+    test "renders body html from markdown" do
+      user = insert(:user)
+      project = insert(:project)
+      changes = Map.merge(@valid_attrs, %{
+        markdown: "A **strong** body",
+        project_id: project.id,
+        user_id: user.id
+      })
+      changeset = Post.changeset(%Post{}, changes)
+      assert changeset.valid?
+      assert changeset |> get_change(:body) == "<p>A <strong>strong</strong> body</p>\n"
+    end
   end
 
-  test "number is auto-sequenced scoped to project" do
-    user = insert(:user)
-    project_a = insert(:project, title: "Project A")
-    project_b = insert(:project, title: "Project B")
+  describe "&create_changeset/2" do
+    test "is valid with valid attributes" do
+      user = insert(:user)
+      project = insert(:project)
+      changeset = Post.create_changeset(%Post{}, %{
+        markdown: "some content",
+        post_type: "issue",
+        title: "some content",
+        project_id: project.id,
+        user_id: user.id,
+        status: "open"
```

Comment:
Need to cleanup the attributes. The status can be provided, but the create changeset ignores the param and simply sets to "open"